lib/helper/JSONResponse.js

```diff
@@ -245,7 +257,11 @@ class JSONResponse extends Helper {
    */
   seeResponseEquals(resp) {
     this._checkResponseReady();
-    expect(this.response.data).to.deep.equal(resp);
+    if (this.response.data.length > 0) {
+      this.response.data.forEach(data => expect(data).to.deep.equal(resp));
```

Comment:
Sorry, can't agree on this.

This checks that data is identical, so in case of array it should be:

```js
I.seeResponseEquals([ a: 'b' ])
```

Even it doesn't work with array, the proposed solution, to check that every element of array equals to provided one won't work on arrays of size > 1. If there are 2 different items it will fail. 

So please remove this one

Reply:
It's done, @DavertMik